### PR TITLE
add git init submodule to docs

### LINKED ISSUE #1483: build.md: after git clone and before releash.sh a "git submodule update --init --recursive" is needed
when following the mayastor build instructions per build.md

there is no mention of:

git submodule update --init --recursive

(no git clone either, don't know if that is a good thing btw)

please update docs, add:

git submodule update --init --recursive

### PR DESCRIPTION
**Pull request by bhuism**
fixes #1483

### CODE REVIEW

**Comment by tiagolobocastro** [comment]
Hi @bhuism could you please change your commit to [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/), example:
`docs: add git init submodule`
Also re require DCO Sign Off, please add a signature, which you can do if you add --signoff to your git command.
So you can fix both with `git commit --amend --signoff`

**Comment by bhuism** [comment]
@tiagolobocastro done

File: doc/build.md [hunk 1]
```diff
@@ -98,6 +98,14 @@ You can use a tool like [`direnv`][direnv] to automate `nix shell` entry.
 If you are unable to use the Nix provided Rust for some reason, there are `norust` and
 `nospdk` arguments to Nix shell. `nix-shell --arg norust true`
 
+After closing the repository don't forget to run a:
```
Comment (tiagolobocastro) [inline doc/build.md]:
```suggestion
After cloning the repository don't forget to run a:
```

**Comment by tiagolobocastro** [comment]
@bhuism please fix the update commit.

**Review by tiagolobocastro — APPROVED** [review]

**Comment by tiagolobocastro** [comment]
@bhuism could you please rebase instead of merging? 
Also you shouldn't need to merge usually, bors uses merges which will create a merge commit and run the tests.

**Comment by bhuism** [comment]
@tiagolobocastro is this what you mean? I rebased my branch from mayastor develop branch, then force pushed it

**Comment by tiagolobocastro** [comment]
> @tiagolobocastro is this what you mean? I rebased my branch from mayastor develop branch, then force pushed it

A single commit would be preferred but this is fine. I'll request for more reviewers now.

**Comment by hrudaya21** [comment]
LGTM. Single commit is preferrable.

**Review by hrudaya21 — APPROVED** [review]

**Comment by tiagolobocastro** [comment]
bors merge

**Comment by tiagolobocastro** [comment]
bors merge

**Comment by tiagolobocastro** [comment]
bors cancel

**Pull request merged**